File: dummySolve.py
def main_YOURNAME(board):
    # fire shots in from top left to bottom right
    for y in range(10):
        for x in range(10):
            board.fireShot(x,y)

            #dont need this but you can use it
                #interface will stop when game is over
            if board.isSolved():
                return

File: java/dummySolve.java
public class dummySolve {
    public static void main(String[] args) {
        System.out.println("hello!");
        // main(board);
    }
    public static String main_YOURNAME(Interface board){
        for (int y = 0; y < 10; y++){
            for (int x = 0; x < 10; x++){
                board.fireShot(x, y);

                if (board.isSolved()){
                    return "exit";
                }
            }
        }
        return "exit";
    }
}
